### PR TITLE
 sniffer: make baudrate optional

### PR DESCRIPTION
**Pull request by miri64**
Depends on #37

### CODE REVIEW

File: sniffer/tools/sniffer.py [hunk 1]
```diff
@@ -135,25 +136,39 @@ def connect(argv):
...
+    socket_p = sp.add_parser("socket",
+                             help="Parse output of a TCP-connected sniffer "
+                                  "application")
+    socket_p.add_argument("host", type=str,
+                          help="Host of the TCP-based sniffer application")
+    socket_p.add_argument("port", type=int,
+                          help="Port of the TCP-based sniffer application")
+    p.add_argument("channel", type=int, help="Channel to sniff on")
+    p.add_argument("outfile", type=str, default=None, nargs="?",
+                   help="PCAP file to output to")
```
Comment (bergzand) [inline sniffer/tools/sniffer.py]: There is `argparse.FileType` to handle file arguments in python.

Comment (miri64) [reply sniffer/tools/sniffer.py]: This change is part of #37. (sorry, did not make clear this PR depends on that)

Comment (miri64) [reply sniffer/tools/sniffer.py]: But I'll have a look!

Comment (bergzand) [reply sniffer/tools/sniffer.py]: Ah, that explains why my comment didn't show up at #37, I meant to place it there :)

**Comment by miri64** [comment]
Rebased.

**Pull request merged**